docs/topics/hacking/debugging.rst

```diff
@@ -0,0 +1,47 @@
+=========
+Debugging
+=========
+
+The :ref:`docker setup <install-with-docker>` uses supervisord to run the
+django runserver. This means if you want to access the management server
+from a shell to run things like ipdb_ you still can.
```

Comment:
Ah here it is: https://github.com/mozilla/marketplace-env/blob/master/docker/base-images/python27/Dockerfile#L14